addons/resource/views/resource_views.xml

```diff
@@ -109,6 +109,7 @@
                         <h1>
                             <field name="name"/>
                         </h1>
+                        <field name="company_id" groups="base.group_multi_company"/>
```

Comment:
Message says it fixes res copany stuff but this changes res company stuff. Please fix res copany and unfix res company.